webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/UserFormInfoPart.ts

```diff
@@ -138,14 +137,6 @@ export class UserFormInfoPart extends FormPart<IUserFormInfoState, IUserFormStat
       }
     }
   }
-  protected override configure() {
-    const loadableStateContext = this.formState.dataContext.get(DATA_CONTEXT_LOADABLE_STATE)!;
-
-    loadableStateContext.getState('user-info', () => [
-      getCachedDataResourceLoaderState(this.serverConfigResource, () => undefined),
-      getCachedDataResourceLoaderState(this.authRolesResource, () => undefined),
-    ]);
-  }
```

Comment:
please check DATA_CONTEXT_LOADABLE_STATE usages and remove if needed

Reply:
did not remove cause it is left in 1 place:
`/Users/stellech/development/dbeaver/cloudbeaver/webapp/packages/core-view/src/Menu/useMenuContext.ts`